### ISSUE TITLE
Error encountered while trying to read file CSV

### ISSUE THREAD

**Issue by hdayacr** [django-import-export/django-import-export#957]
Hi, 

Django==1.8
django-import-export==1.1.0

When I want to import a small csv (1 line) I have directly a blank page with the message 

> Error encountered while trying to read file: petit.csv 

I see in the network console a POST to http://localhost:8001/admin/XXX/XXX/import/ 
with a 200 response with this message.

It is the same behavior whatever the model.

**Comment by andrewgy8** [comment]
Hey @hdayacr ! Unfortunately there is not enough info here for me to diagnose the problem. 

Does the one line include headers? What does the csv look like? Is it comma or semi-colon separated? Can you try with `django-import-export==1.2.0`? What do your resource classes look like?

**Comment by hdayacr** [comment]
Sorry, there is more infos :

```
# -*- coding: utf-8 -*-
from import_export import resources
from XX.models.xxx import XXX

class XXXResource(resources.ModelResource):
    """
    This class handle Django Admin import and export module
    """
    class Meta:
        model = XXX
        fields = ('id', 'name', 'nothing')
```
The csv is exactly 
```
id,name,nothing
16684,blabla,toto
```

I tried with django-import-export==1.2.0, nothing change

**Comment by hdayacr** [comment]
I got the project locally and comment the lines 286 and 287 from import_export/admin.py
```
except Exception as e:
            return HttpResponse(_(u"<h1>%s encountered while trying to read file: %s</h1>" % (type(e).__name__, import_file.name)))
```
So there is the stackTrace of the problem

> Traceback:
> File "/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
>   132.                     response = wrapped_callback(request, *callback_args, **callback_kwargs)
> File "/usr/local/lib/python2.7/site-packages/django/utils/decorators.py" in _wrapped_view
>   110.                     response = view_func(request, *args, **kwargs)
> File "/usr/local/lib/python2.7/site-packages/django/views/decorators/cache.py" in _wrapped_view_func
>   57.         response = view_func(request, *args, **kwargs)
> File "/usr/local/lib/python2.7/site-packages/django/contrib/admin/sites.py" in inner
>   233.             return view(request, *args, **kwargs)
> File "/usr/local/lib/python2.7/site-packages/import_export/admin.py" in import_action
>   283.                 dataset = input_format.create_dataset(data)
> File "/usr/local/lib/python2.7/site-packages/import_export/formats/base_formats.py" in create_dataset
>   148.             return super(CSV, self).create_dataset(in_stream.encode('utf-8'), **kwargs)
> File "/usr/local/lib/python2.7/site-packages/import_export/formats/base_formats.py" in create_dataset
>   105.         self.get_format().import_set(data, in_stream, **kwargs)
> File "/usr/local/lib/python2.7/site-packages/tablib/formats/_csv.py" in import_set
>   38.     for i, row in enumerate(rows):
> File "/usr/local/lib/python2.7/site-packages/backports/csv.py" in __next__
>   409.                 raise Error(err_str.format(typ_name))
> 
> Exception Type: Error at /admin/XX/XXX/import/
> Exception Value: iterator should return strings, not bytes (did you open the file in text mode?)
>

**Comment by hdayacr** [comment]
I fixed the problem by adding in "requirements.txt" file 
`tablib==0.12.1`

**Comment by hdayacr** [comment]
Fixed #959

**Comment by mohit-jft** [comment]
getting error ValueError encountered while trying to read file: users(1).xlsx
